### DISCUSSION TITLE
Add checkbox to allow computerUse with other models

### DISCUSSION THREAD

**Discussion by migmac99** [cline/cline#874] [Feature Requests]
I would like to be able to experiment with computer use with my local Ollama models. I have a few that might work and would like the ability to test this as currently computer use is hardcoded to only be allowed for claude. It would be awesome to have an override setting to experiment at will

**Comment by Achim-Sommer** [comment]
I think that's because Computer Use is a function of Claude and not of Cline

**Reply by migmac99** [reply]
Yes but I'd still like to be able to experiment.
